### ISSUE TITLE
storage location sub field

### ISSUE THREAD

**Issue by DIYsciBorg** [partkeepr/PartKeepr#413]
Most people have at least one rack of bins. These bins only hold one or two part items (or a few?). So, having something like "rack 01, bin 01.  rack 01, bin02...." can get tedious.
In addition, doing an inventory count check is most easily accomplished by using the view location panel, and seeing everything/adjusting everything in that location at once.

So, I suggest a sub-field for the storage location of the part data, which is somewhat meaningless as far as the "view location" panel is concerned. Simply set "rack 01" as the primary field and "01" as the sub field.
Now, when viewing the "Rack 01" location, you have all the bins in the rack displayed with their contents, so you can quickly check and adjust your inventory.

**Comment by felicitus** [comment]
Would #225 help you? If so, may I close the issue and mark it as duplicate?

**Comment by DIYsciBorg** [comment]
Uhh  very likely. Provided that when looking the parent, you have the ability to see all the sub location inventory. My vision of the application of this is doing inventory counts / updates / adjustments on a monthly basis. When each drawer only holds one inventory item, its tedious to go drawer by drawer.
If the plan for #225 makes seeing the whole group's inventory in a table, then yeah, close this one.

**Comment by felicitus** [comment]
Yes, you're right. I'll leave this open to remind me when one filters by storage location, that there should be a checkbox "display contents of sub-storage locations" as well. Or to add separate reports, I'm not sure yet.
